app/views/cms/members/_menu.html.erb

```diff
@@ -1,31 +1,31 @@
 <nav class="nav-menu">
   <% if params[:action] =~ /index/ %>
     <% if @model.allowed?(:edit, @cur_user, site: @cur_site, node: @cur_node) %>
-      <%= link_to :new, { action: :new }, class: :new %>
+      <%= link_to t('ss.links.new'), { action: :new }, class: :new %>
     <% end %>
     <% if @model.allowed?(:read, @cur_user, site: @cur_site, node: @cur_node) %>
-      <%= link_to :download, { action: :download }.merge(s: params[:s]), class: :download %>
+      <%= link_to t('ss.links.download'), { action: :download }.merge(s: params[:s]), class: :download %>
```

Comment:
__[brakeman]__
* Unsafe parameter value in link_to href [:ref](http://brakemanscanner.org/docs/warning_types/link_to_href)

<!-- Comment by [SideCI](https://sideci.com) -->